config/overlays/env/kustomization.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Can this file gets merged to config/default/kustomization.yaml? not sure if we want this overlay, we can create an overlay for kubeflow though.

Reply:
I see, moved.